src/tools/annotation/RectangleRoiTool.js

```diff
@@ -457,7 +477,9 @@ function _formatArea(area, hasPixelSpacing) {
     ? ` mm${String.fromCharCode(178)}`
     : ` px${String.fromCharCode(178)}`;
 
-  return `A: ${toGermanNumberStringTemp(area)} ${suffix}`; //`Area: ${numbersWithCommas(area.toFixed(2))}${suffix}`;
+  // TODO: Localisation!!
+  return `A: ${area} ${suffix}`;
+  // `A: {toGermanNumberStringTemp(area)} ${suffix}`;
```

Comment:
Maybe this comment is old code? Can we remove it?

Reply:
Yes, since we will create new localisation, I will remove these comments